### PR TITLE
Alex/feature/user levels

### PR DESCRIPTION
**Pull request by AlexandrBelenov**
We've developed a mechanism of access to courses based on a user's roles.

### CODE REVIEW

File: app/logger.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import logging
+import sys
+
+LOGGER_NAME = 'AVANTIKA'
```
Comment (kipod) [inline app/logger.py]: PRACTICE_CODING

**Review by LikaDevelop — APPROVED** [review]
Everything is fine.

File: app/tab/utils.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+
+def load_tabs():
+    '''
+    It loads a config tabs.json and parses it to an array of app.tab.models.Tab.
+    The config tabs.json have to be near the app module.
+    '''
+
+    config_path = os.path.abspath(CONFIG_TABS)
+    log(log.INFO, "Tabs config file path: {}".format(config_path))
```
Comment (kipod) [inline app/tab/utils.py:37]: log(log.INFO, "Tabs config file path: %s", config_path)

File: app/tab/utils.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    tab_objects = []
+    try:
+        with open(config_path, "r") as f:
+            for tab in json.load(f):
+                tab_objects.append(Tab.from_dict(**tab))
+    except OSError as ose:
+        log(log.ERROR, "OS error: {}".format(ose))
```
Comment (kipod) [inline app/tab/utils.py:44]: use %s syntax

**Review by kipod — COMMENTED** [review]
Good job!

**Pull request merged**